src/zkp/pimod.rs

```diff
@@ -437,7 +450,17 @@ fn y_prime_combinations(
 mod tests {
     use super::*;
     use crate::paillier::prime_gen;
-    use rand::{rngs::OsRng, RngCore};
+    use rand::{
+        rngs::{OsRng, StdRng},
+        Rng, RngCore, SeedableRng,
+    };
+
+    fn rng() -> StdRng {
+        let mut seeder = OsRng;
+        let seed = seeder.gen();
+        eprintln!("seed: {:?}", seed);
+        StdRng::from_seed(seed)
+    }
```

Comment:
note: This will probably go away when we do #28 fully (e.g. it'll get centralized in `lib.rs` or someplace).